app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -101,22 +133,26 @@ def parse_az(az)
       end
 
       def parse_instance(instance)
-        uid         = "#{@subscription_id}\\#{instance.fetch_path('resourceGroup')}\\#{instance.fetch_path('name')}"
+        uid = resource_uid(@subscription_id,
+                           instance.fetch_path('resourceGroup'),
+                           instance.fetch_path('type').downcase,
+                           instance.fetch_path('name'))
```

Comment:
@bzwei - The UID of the instance is going to change, we are planning on using the instance ID property obtained from the model view (as opposed to instance view). This can be a separate PR. However, I see you are including the instance "type", what are the possible instance "types"?